go.mod

```diff
@@ -2,4 +2,4 @@ module k8s.io/klog/v2
 
 go 1.13
 
-require github.com/go-logr/logr v0.4.0
+require github.com/go-logr/logr v1.0.0-rc1
```

Comment:
you have v1.0.0 now https://github.com/go-logr/logr/releases/tag/v1.0.0

Reply:
updated